packages/gafl-webapp-service/src/pages/summary/contact-summary/__tests__/contact-summary.spec.js

```diff
@@ -135,108 +118,53 @@ describe('The contact summary page', () => {
     process.env.ADDRESS_LOOKUP_KEY = 'bar'
     fetch.mockImplementationOnce(async () => new Promise(resolve => resolve({ json: () => searchResultsOne, ok: true })))
+    const data = await injectWithCookies('GET', ADDRESS_LOOKUP.uri)
+    expect(data.payload.search(backLinkRegEx(CONTACT_SUMMARY.uri)) > 0).toBeTruthy()
     await postRedirectGet(ADDRESS_LOOKUP.uri, { premises: 'Howecroft Court', postcode: 'BS9 1HJ' })
     await injectWithCookies('GET', ADDRESS_SELECT.uri)
-    const data = await postRedirectGet(ADDRESS_SELECT.uri, { address: '0' })
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(CONTACT_SUMMARY.uri)
+    const data2 = await postRedirectGet(ADDRESS_SELECT.uri, { address: '0' })
+    expect(data2.statusCode).toBe(302)
+    expect(data2.headers.location).toBe(CONTACT_SUMMARY.uri)
   })
 
   it('address entry amendment causes redirect the summary page', async () => {
-    const data = await postRedirectGet(ADDRESS_ENTRY.uri, goodAddress)
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(CONTACT_SUMMARY.uri)
-    const data2 = await injectWithCookies('GET', CONTACT_SUMMARY.uri)
-    expect(data2.statusCode).toBe(200)
-  })
-
-  it('contact amendment (email) causes redirect the summary page', async () => {
-    const data = await postRedirectGet(CONTACT.uri, { 'how-contacted': 'email', email: 'new3@example.com' })
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(CONTACT_SUMMARY.uri)
-  })
-
-  it('newsletter amendment causes redirect the contact summary page', async () => {
-    const data = await postRedirectGet(NEWSLETTER.uri, { newsletter: 'yes', email: 'example2@email.com' })
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(CONTACT_SUMMARY.uri)
-  })
-
-  it('date of birth amendment causes redirect the licence summary page', async () => {
-    await injectWithCookies('POST', DATE_OF_BIRTH.uri, dobHelper(dobAdultToday))
-    const data = await injectWithCookies('GET', CONTROLLER.uri)
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(LICENCE_SUMMARY.uri)
-  })
-
-  it('date of birth (senior) amendment causes redirect the licence summary page and then the contact summary page', async () => {
-    await injectWithCookies('POST', DATE_OF_BIRTH.uri, dobHelper(dobSeniorToday))
-    const data = await injectWithCookies('GET', CONTROLLER.uri)
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(LICENCE_SUMMARY.uri)
-    await injectWithCookies('GET', LICENCE_SUMMARY.uri)
-    await injectWithCookies('POST', LICENCE_SUMMARY.uri)
-
-    const data2 = await injectWithCookies('GET', CONTROLLER.uri)
+    const data = await injectWithCookies('GET', ADDRESS_ENTRY.uri)
+    expect(data.payload.search(backLinkRegEx(CONTACT_SUMMARY.uri)) > 0).toBeTruthy()
+    const data2 = await postRedirectGet(ADDRESS_ENTRY.uri, goodAddress)
```

Comment:
Have all the redirects on the left stopped happening?

Reply:
I moved a lot of the tests regards dob from the contact to the licence summary. (I forgot to move them before - originally the dob was on the licence summary)